src/orange/surf/Involute.hh

```diff
@@ -98,7 +98,10 @@ class Involute
     CELER_FUNCTION real_type r_b() const { return std::fabs(r_b_); }
 
     //! Displacement angle
-    CELER_FUNCTION real_type a() const { return a_; }
+    CELER_FUNCTION real_type displacement_angle() const
+    {
+        return displacement_angle_;
```

Comment:
All I really care about is the public accessors for these (what you'd see from someone calling this class): you can leave the internal implementation as `a_` since it's well described internally.